contracts/oracles/IporOracle.sol

```diff
@@ -165,16 +158,25 @@ contract IporOracle is
         uint256 length = indexesToUpdate.length;
         require(length > 0, IporErrors.INPUT_ARRAYS_LENGTH_MISMATCH);
```

Comment:
This validation is redundant - it had sense when we have two arrays in input param, now we have only one array with structure.